crypto/x509/x509_vfy.c

```diff
@@ -2374,6 +2576,13 @@ void X509_STORE_CTX_set0_crls(X509_STORE_CTX *ctx, STACK_OF(X509_CRL) *sk)
     ctx->crls = sk;
 }
 
+#ifndef OPENSSL_NO_OCSP
+void X509_STORE_CTX_set0_ocsp_resp(X509_STORE_CTX *ctx, STACK_OF(OCSP_RESPONSE) *sk)
```

Comment:
This function should be actually named X509_STORE_CTX_set_ocsp_resp (not with set0) as the store ctx does not take the ownership of the stack. Please rename it and mention the semantics in the documentation. I.e., that the stack must outlive the X509_STORE_CTX.